### PR TITLE
Implements Technical Metadata. 

### PR DESCRIPTION
**Pull request by hectorcorrea**
This is to address #16

### CODE REVIEW

File: hydra-pcdm.gemspec [hunk 1]
```diff
@@ -20,6 +20,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_dependency 'activefedora-aggregation', '~> 0.2.0'
+  spec.add_dependency 'rdf-vocab', '~> 0.8.1'
```
Comment (hectorcorrea) [inline hydra-pcdm.gemspec]: ignore this.

**Comment by hectorcorrea** [comment]
@escowles ping

File: lib/hydra/pcdm/models/file.rb [hunk 2]
```diff
@@ -5,6 +5,16 @@ class File < ActiveFedora::File
...
+
+      property :file_name, predicate: EBUCoreVocabularies::EBUCoreTerms.filename 
+      property :file_size, predicate: EBUCoreVocabularies::EBUCoreTerms.fileSize 
+      property :date_created, predicate: EBUCoreVocabularies::EBUCoreTerms.dateCreated
+      property :has_mime_type, predicate: EBUCoreVocabularies::EBUCoreTerms.hasMimeType
+      property :date_modified, predicate: EBUCoreVocabularies::EBUCoreTerms.dateModified
+      property :byte_order, predicate: SweetjplVocabularies::SweetjplTerms.byteOrder
+
+      # This is a server-managed predicate which means Fedora does not let us change it.
+      property :file_hash, predicate: RDF::Vocab::PREMIS.hasMessageDigest
```
Comment (escowles) [inline lib/hydra/pcdm/models/file.rb]: In addition to the digest, there are five other server-managed properties: fedora:created, fedora:lastModified, fedora:mimeType, premis:hasSize and premis:hasOriginalName (so everything here but byte-order).
fedora:mimeType and premis:hasOriginalName can be set in the initial request to create the file, but the other four are completely up to fedora.  Do we want to provide read-only support for any of those other properties?

Comment (hectorcorrea) [inline lib/hydra/pcdm/models/file.rb]: For those reading at home...Esme will run this through a larger group via Hydra Tech. The PR is OK as-is for now (once the build passes)

Comment (hectorcorrea) [inline lib/hydra/pcdm/models/file.rb]: Here is the question that Esme posted to Hydra-Tech https://groups.google.com/forum/#!topic/hydra-tech/EEVV_IjGMlY

Comment (escowles) [inline lib/hydra/pcdm/models/file.rb]: Also, I've opened PRs to use premis:hasMessageDigest for the SHA1 digest, and to make the mimeType and filename modifiable:

https://github.com/fcrepo4/fcrepo4/pull/790
https://github.com/fcrepo4/fcrepo4/pull/791

We'll talk about this at today's tech metadata call: https://wiki.duraspace.org/display/hydra/Technical+Metadata+Call+2015-05-12

**Comment by jcoyne** [comment]
@hectorcorrea why is dino.jpg in this pr?

**Comment by hectorcorrea** [comment]
@jcoyne I was using the file as a content at one point but not anymore...plus I like dinos :) I'll remove the file and resubmit.

**Comment by hectorcorrea** [comment]
@jcoyne dino is now extinct :boom:

**Pull request merged**